### PR TITLE
Fix wifi_getRadioEnable

### PR DESCRIPTION
**Pull request by yanhou-syna**
Currently it sets default to be disabled. Webui shows both WiFi are grey out.

Removed the default value and tested to see both WiFi are enabled.

### CODE REVIEW

File: source/wifi/wifi_hal.c [hunk 1]
```diff
@@ -1032,8 +1032,6 @@ INT wifi_getRadioEnable(INT radioIndex, BOOL *output_bool)      //RDKB
                *output_bool=TRUE;
 
         //TODO: check if hostapd with config is running
-
-        *output_bool = FALSE;
```
Comment (manigandanj) [inline source/wifi/wifi_hal.c]: Can you please move this line to line 1028 ie before _sprintf(cmd,"ifconfig %s%d | grep RUNNING ", AP_PREFIX, radioIndex);_

**Review by manigandanj — CHANGES_REQUESTED** [review]
Hello @yanhou-syna , please address my comment.

File: source/wifi/wifi_hal.c (hunk 1 above)
Comment (yanhou-syna) [reply source/wifi/wifi_hal.c]: Moved a little bit earlier to cover the error case as well

**Comment by manigandanj** [comment]
Looks okay to me.

**Review by rajkumar154 — APPROVED** [review]

**Review by rajkumar154 — DISMISSED** [review]

**Pull request closed without merging**